tanner/emulators/cmd_exec.py

```diff
@@ -1,20 +1,15 @@
-from tanner.utils import docker_helper
+from tanner.utils import aiodocker_helper
 from tanner.utils import patterns
 
 
 class CmdExecEmulator:
     def __init__(self):
-        self.helper = docker_helper.DockerHelper()
+        self.helper = aiodocker_helper.AIODockerHelper()
 
-    async def create_attacker_env(self, session):
```

Comment:
why you don't associate docker with a session anymore?

Reply:
For the above reason, I only followed one method to execute commands (`containers.run`) :) so haven't used a container like the previous implementation.

Reply:
You're right, got your point.